pkg/operation/botanist/component/kubeapiserver/deployment.go

```diff
@@ -1281,3 +1274,129 @@ func (k *kubeAPIServer) handleWatchdogSidecar(deployment *appsv1.Deployment, con
 		},
 	})
 }
+
+func (k *kubeAPIServer) handleAuditSettings(deployment *appsv1.Deployment, configMapAuditPolicy *corev1.ConfigMap, secretWebhookKubeconfig *corev1.Secret) {
+	deployment.Spec.Template.Spec.Containers[0].Command = append(deployment.Spec.Template.Spec.Containers[0].Command, fmt.Sprintf("--audit-policy-file=%s/%s", volumeMountPathAuditPolicy, configMapAuditPolicyDataKey))
+
+	deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
+		Name:      volumeNameAuditPolicy,
+		MountPath: volumeMountPathAuditPolicy,
+	})
+	deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes, corev1.Volume{
+		Name: volumeNameAuditPolicy,
+		VolumeSource: corev1.VolumeSource{
+			ConfigMap: &corev1.ConfigMapVolumeSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: configMapAuditPolicy.Name,
+				},
+			},
+		},
+	})
+
+	if k.values.Audit == nil || k.values.Audit.Webhook == nil {
+		deployment.Spec.Template.Spec.Containers[0].Command = append(deployment.Spec.Template.Spec.Containers[0].Command,
+			"--audit-log-path=/var/lib/audit.log",
```

Comment:
`--audit-log-path` is only specified, if there is no `Audit` configuration.
According to the [Kubernetes documentation](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#log-backend), this deactivates the log backend in any other cases. Is this intended?

Reply:
Yes